vehicle/raw_vehicle_cmd_converter/include/raw_vehicle_cmd_converter/accel_map.hpp

```diff
@@ -29,12 +29,12 @@ namespace raw_vehicle_cmd_converter
 class AccelMap
 {
 public:
-  bool readAccelMapFromCSV(std::string csv_path);
-  bool getThrottle(double acc, double vel, double & throttle);
-  bool getAcceleration(double throttle, double vel, double & acc);
-  std::vector<double> getVelIdx() { return vel_index_; }
-  std::vector<double> getThrottleIdx() { return throttle_index_; }
-  std::vector<std::vector<double>> getAccelMap() { return accel_map_; }
+  bool readAccelMapFromCSV(const std::string & csv_path);
+  bool getThrottle(const double acc, double vel, double & throttle) const;
```

Comment:
I prefer returning optional<double> instead of bool and referenced double.
Bust this is just my personal opinion, so you don't have to fix.

Reply:
this will change IF for other packages too So I remain unchanged this with this PR